ingestion/setup.py

```diff
@@ -95,6 +95,7 @@ def get_long_description():
     "Jinja2>=2.11.3",
     "jsonpatch==1.32",
     "jsonschema",
+    "memory-profiler",
```

Comment:
adding this by default, would help us debug similar scenarios later on with existing builds. It does not add any extra dependency